src/server/services/wmo/wmo.ts

```diff
@@ -60,6 +61,19 @@ function addDocumentLinksToLineItems(
   });
 }
 
+function getLatestStatus(steps: StatusLineItem[]) {
+  if (!steps.length) {
+    return 'Onbekend';
+  }
+  const mostRecentStep = steps.reduce((acc, step) =>
+    isAfter(new Date(step.datePublished), new Date(acc.datePublished))
+      ? step
+      : acc
+  );
+
+  return mostRecentStep.status;
```

Comment:
```suggestion
return steps.find((step)=>step.isActive)?.status ?? 'Onbekend'
```

Reply:
ik mag ervan uitgaan dat datePublished aflopend gesorteerd is? altijd

Reply:
Je mag er vanuit gaan dat er maar één actieve status is.